jdaviz/configs/cubeviz/helper.py

```diff
@@ -1,12 +1,18 @@
 import numpy as np
+from packaging.version import Version
 
 from glue.core import BaseData
+from glue_jupyter import __version__ as gj_version
 from jdaviz.core.helpers import ImageConfigHelper
 from jdaviz.configs.default.plugins.line_lists.line_list_mixin import LineListMixin
 from jdaviz.configs.specviz import Specviz
 from jdaviz.core.events import (AddDataMessage,
                                 SliceSelectSliceMessage)
 
+# NOTE: this and the if-statement that uses it can be removed if/once
+# the version of glue-jupyter with as_steps support is pinned as min-version
+GLUEJUPYTER_HAS_AS_STEPS = Version(gj_version) >= Version('1.3.0')
```

Comment:
This is the syntax used by Astropy. I think it is easier to grep for as we bump minversion. And in case we also need to use this check for something else.
```suggestion
GLUEJUPYTER_GE_1_3 = Version(gj_version) >= Version('1.3.0')
```

Reply:
This check does fail if 1.3 isn't released yet because 1.3.dev is compared differently, which gives me pain every time.

Also I wonder if the syntax is nicer if you use `astropy.utils.introspection.minversion()`. Under the hood, it is the same comparison, but you only have to pass in package name, not the version.

Reply:
I did `HAS_AS_STEPS` to make the reason more clear and remind us to remove it.  But I have no problem changing it if that is a standard astropy syntax (and we have the comment stating its current purpose to hopefully remind us to remove it if there isn't any other cause for it).  I'll look into minversion and see if that's simpler and change that too.

Regarding 1.3.dev... that's true and does make testing this change a pain, but apart from hardcoding the current bugfix release version of glue-jupyter or manually accounting for the dev version in the string, I'm not sure what to do about that.  I guess we could say that we're not responsible for manual installs of upstream packages since that wouldn't necessarily have to obey the version pins either?

Reply:
updated to your suggested syntax and to use `minversion` to avoid the glue_jupyter import.  Thanks!